framework-plugins/lisk-framework-dashboard-plugin/src/ui/components/widgets/MyAccountWidget.tsx

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright © 2021 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+
+import * as React from 'react';
+import { Widget, WidgetBody, WidgetHeader } from '../widget';
+import { TableBody, TableHeader, Table } from '../Table';
+import CopiableText from '../CopiableText';
+import Text from '../Text';
+
+interface AccountInfo {
+	readonly binaryAddress?: string;
+	readonly publicKey?: string;
+}
+
+interface MyAccountProps {
+	accounts: ReadonlyArray<AccountInfo>;
+	onSelect?: () => void;
```

Comment:
It seems `onSelect` is not passing any value to handler, so how will parent component know which account was clicked?